week1/homework/src/server.js

```diff
@@ -1,20 +1,37 @@
-'use strict';
+'use strict'
 
-const http = require('http');
+const http = require('http')
 
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
-function createServer(port) {
-  let state = 10;
+function createServer () {
+  let state = 10
 
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
-  });
+  const server = http.createServer((req, res) => {
+    if (req.url == '/state') {
+      res.writeHead(200, { 'Content-Type': 'application/json' })
+      res.write(JSON.stringify({ state: state }))
```

Comment:
the 200 header plus the write is repeating, how could you remove some of the duplications?